Simplenote/Resources/release_notes.txt

```diff
@@ -1 +1,2 @@
-Added preferences menu to centralize app options and customizations.
+- Fixed an issue with horizontal clipping where the last character at the end of the line wasn't displayed completel
```

Comment:
Seems like there is a spelling issue in `completel`. I assume it's meant to be `completely`?

Reply:
Spot on. Thanks!